src/graph_compute/common.h

```diff
@@ -75,17 +92,6 @@
   #define cilk_sync
 #endif
 
-typedef uint64_t vid_t;  // vertex id type
-
-struct vertex_t {
-  vid_t id;
-  vid_t priority;
-  vid_t dependencies;
-  volatile vid_t satisfied;
-  vid_t cntEdges;
-  vid_t * edges;
-  double data;
-};
-typedef struct vertex_t vertex_t;
+//  typedef uint32_t vid_t;  // vertex id type
```

Comment:
`vid_t` has no definition -- this one is commented out, and there is no other one. Also, why did the type change from `uint64_t` to `uint32_t`?